### DISCUSSION TITLE
Does this project work with a dedicated server bought from hetzner auction?

### DISCUSSION THREAD

**Discussion by elieobeid7** [kube-hetzner/terraform-hcloud-kube-hetzner#640] [Q&A] [answered]
Does this project work with a dedicated server bought from hetzner auction? Or is it specifically made to be used on hetzner cloud platform?

**Comment by mysticaltech** [comment]
Not in its natural form because these servers are not exposed in Hetzner Cloud via the hcloud terraform library, but some users have been able to make it work. Please research the project, you will see how they did it.

**Reply by Elliott-Green** [reply]
Hi @mysticaltech 

I've researched this topic and haven't found any leads on it either in discussions, issues, or google.
Are you able to remember where you've seen this working?

**Comment by mysticaltech** [comment, marked as answer]
@Elliott-Green Here you go! https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/issues/433
